linkerd/trace-context/src/service.rs

```diff
@@ -35,20 +36,35 @@ impl<K: Clone, S> TraceContext<K, S> {
     }
 
     fn request_labels<B>(req: &http::Request<B>) -> HashMap<&'static str, String> {
```

Comment:
```suggestion
    /// Returns labels for the provided request.
    ///
    /// The OpenTelemetry spec defines the semantic conventions that HTTP services should use for the labels included in traces: https://opentelemetry.io/docs/specs/semconv/http/http-spans/
    fn request_labels<B>(req: &http::Request<B>) -> HashMap<&'static str, String> {
```

it might be nice to add a note documenting where the spec we are following lives.

**nb:** this suggestion will need to be formatted, sorry :see_no_evil: